plugins/vestige/vestige.cpp

```diff
@@ -558,33 +628,62 @@ VestigeInstrumentView::VestigeInstrumentView( Instrument * _instrument,
 	m_rolRPresetButton->setShortcut( Qt::Key_Plus );
 
 
-	m_selPresetButton = new QPushButton( tr( "" ), this );
-	m_selPresetButton->setGeometry( 228, 201, 16, 16 );
-
 	QMenu *menu = new QMenu;
-
 	connect( menu, SIGNAL( aboutToShow() ), this, SLOT( updateMenu() ) );
 
-
-	m_selPresetButton->setIcon( embed::getIconPixmap( "stepper-down" ) );
-
+	m_selPresetButton = new QPushButton("", this);
+	m_selPresetButton->setGeometry(54, 180, 16, 16);
+	m_selPresetButton->setIcon(embed::getIconPixmap( "stepper-down" ));
 	m_selPresetButton->setMenu(menu);
 
 
-	m_toggleGUIButton = new QPushButton( tr( "Show/hide GUI" ), this );
-	m_toggleGUIButton->setGeometry( 20, 130, 200, 24 );
-	m_toggleGUIButton->setIcon( embed::getIconPixmap( "zoom" ) );
-	m_toggleGUIButton->setFont( pointSize<8>( m_toggleGUIButton->font() ) );
-	connect( m_toggleGUIButton, SIGNAL( clicked() ), this,
-							SLOT( toggleGUI() ) );
-
-	QPushButton * note_off_all_btn = new QPushButton( tr( "Turn off all "
-							"notes" ), this );
-	note_off_all_btn->setGeometry( 20, 160, 200, 24 );
-	note_off_all_btn->setIcon( embed::getIconPixmap( "stop" ) );
-	note_off_all_btn->setFont( pointSize<8>( note_off_all_btn->font() ) );
-	connect( note_off_all_btn, SIGNAL( clicked() ), this,
-							SLOT( noteOffAll() ) );
+	m_toggleGUIButton = new PixmapButton( this, "" );
+	m_toggleGUIButton->setGeometry( 20, 130, 20, 20 );
+	m_toggleGUIButton->setActiveGraphic( embed::getIconPixmap(
+							"zoom", 20, 20  ) );
+	m_toggleGUIButton->setInactiveGraphic( embed::getIconPixmap(
+							"zoom", 20, 20  ) );
+	connect( m_toggleGUIButton, SIGNAL( clicked() ),
+		 this, SLOT( toggleGUI() ) );
+
+	m_panicButton = new PixmapButton( this, "" );
+	m_panicButton->setGeometry( 50, 130, 20, 20 );
+	m_panicButton->setActiveGraphic( embed::getIconPixmap(
+							"stop", 20, 20  ) );
+	m_panicButton->setInactiveGraphic( embed::getIconPixmap(
+							"stop", 20, 20  ) );
+	connect( m_panicButton, SIGNAL( clicked() ),
+		 this,SLOT( noteOffAll() ) );
+
+	m_capturePgmChangeButton = new PixmapButton(this,
+						  tr("Capture Program Change events"));
+	m_capturePgmChangeButton->setActiveGraphic(embed::getIconPixmap("led_green"));
+	m_capturePgmChangeButton->setInactiveGraphic(embed::getIconPixmap("led_off"));
+	m_capturePgmChangeButton->setCheckable(true);
+	ToolTip::add( m_capturePgmChangeButton, tr(
```

Comment:
The tooltip text looks too long to me.

Reply:
Reworked the solution, so the control does not exist anymore, neither does the tooltip.